client/src/Screen/Login/LoginPage.tsx

```diff
@@ -0,0 +1,203 @@
+                <div className="firstbox">
+                  <Typography
+                    variant={"h4"}
+                    style={{
+                      margin: 4,
+                      fontFamily: "Edu AU VIC WA NT Arrows ,cursive",
+                      position: "relative",
+                      top: "145px",
+                      left: "75px",
+                      fontSize: "55px",
+                      width: "268px",
+                    }}
+                  >
+                    Tip /Mart
+                  </Typography>
+                </div>
+              </Box>
+              <Box>
+              
+    
+
+                <Typography
+                  sx={{
+                    position: "absolute",
+                    left: "70%",
+                    mt: 2,
+                    fontFamily: "Dancing Script, serif",
+                    cursor: "pointer",
+                    color:"blue",
+                    padding: "3px",
+                    top:" 334px",
+                    width: "257px;",
+                    textAlign: "center",
+                    borderRadius: "5px",
+                    fontSize: "19px",
+                    zIndex:"1",
+                    // textDecoration:"underline",
+                    fontStyle:"italic",
+  
+                  }}
+                  fontFamily={ "fantasy"}
+                  component={"h5"}
+                 
+                >
+                  <Typography variant="subtitle1" sx={{display:"inline" ,color:"black",textDecoration:"none"}}>Dont have account ? </Typography>
+                   Sign-up 
+                  
+                </Typography>
+                <Link to="/">
+                <CloseIcon sx={{position:"absolute",zIndex:1,right:"0px",bottom:"91%",fontSize:"37px",color:"white",background:"red",borderRadius:"3px"}}></CloseIcon>
+                </Link>
+                <Box
+                  component={"form"}
+                  onSubmit={loginCoustomer}
+                  sx={{
+                    display: "flex",
+                    justifyContent: "center",
+                    alignItems: "",
+                    height: "100%",
+                    flexDirection: "column",
+                    
+                        backdropFilter: "blur(6px)",
+                        maxWidth: "101%",
+                      pr: "66px"
+                  }}
+                >
+                  <Input
+                    type="email"
+                    label="Email Id"
+                    style={{ mb: 1, width: 300 }}
+                    name="loginEmail"
+                    emailid={inputLogin}
+                  />
+                  <br></br>
+                  <Input
+                    type="password"
+                    label="Password"
+                    style={{ mt: 1, width: 300 }}
+                    emailid={inputLogin}
+                    name="loginPassword"
+                  />
+                  <br></br>

```

Comment:
The use of br tag is wrong check the syntax and better go with box when using MUI.